pyrogram/client/ext/base_client.py

```diff
@@ -75,15 +76,16 @@ def __init__(self):
         self.download_queue = Queue()
         self.download_workers_list = []
 
-    def send(self, data: Object):
-        pass
+    @abstractmethod
+    def send(self, data: Object): pass
```

Comment:
Function body in the same line? Hm I don't quite like it in python, even if is just `pass`

Reply:
Pretty sure that's standard for @abstractmethods